### DISCUSSION TITLE
How to get the current replication value for a volume?

### DISCUSSION THREAD

**Discussion by eliphatfs** [seaweedfs/seaweedfs#5651] [Q&A]
I reconfigured replication of the cluster during operation with volume.configure.replication and volume.fix.replication. However, 150 out of 9000 volumes only have 1 replication now, and is read-only. And nothing happens when running volume.fix.replication. I wonder how the replication is currently set for the volumes and why they are not replicated. However, I did not find a helpful command for this.

**Comment by chrislusf** [comment]
`volume.list` should show the replication type for each volume.

**Reply by eliphatfs** [reply]
Is it 'replica_placement'? And does '1' mean '001'?

**Reply by chrislusf** [reply]
yes
